client/termdb/tree.js

```diff
@@ -201,8 +201,10 @@ class TdbTree {
 					copy.terms = t0.terms
 				}
 			}
+
 			this.termsById[copy.id] = copy
 		}
+		if (terms) for (const child of terms) child.parent = term
```

Comment:
since this affects all mass plots, please comment on purpose

Reply:
I added a comment in the code. Having areference to the parent allows me to identify the domain terms and render the tree in this way

<img width="356" alt="Screenshot 2024-07-25 at 3 36 55 PM" src="https://github.com/user-attachments/assets/b18caa47-842a-4443-879f-77953baed9b2">

Reply:
unfortunately it's circular ref and breaks tests http://localhost:3000/testrun.html?dir=termdb&name=tree.integration